packages/cosmic-swingset/bin/ag-nchainz

```diff
@@ -215,23 +216,23 @@ start-relayer)
     echo "Replacing path \`$path'"
     path=$template
     jq -s "$jqexpr" \
-      "nchainz/config/$template.json" "$tmpfile" > "nchainz/config/$path.json"
+      "$PATHCONFIG/$template.json" "$tmpfile" > "$PATHCONFIG/$path.json"
   else
     # Create a new path.
     i=0
     path=path-$i
-    while [[ -e "nchainz/config/$path.json" ]]; do
+    while [[ -e "$PATHCONFIG/$path.json" ]]; do
       i=$(( $i + 1 ))
       path=path-$i
     done
 
     echo "Creating new path \`$path'"
     jq -s "$jqexpr"' * { "generator": "ag-nchainz" }' \
-      "nchainz/config/$template.json" "$tmpfile" > "nchainz/config/$path.json"
+      "$PATHCONFIG/$template.json" "$tmpfile" > "$PATHCONFIG/$path.json"
   fi
 
-  echo "$ rly config add-dir nchainz/config/"
-  rly config add-dir nchainz/config/
```

Comment:
Is this change prompted by a corresponding upstrteam change in nchainz?

I think I found upstream source for nchainz earlier, but I'm struggling to find it now.

Reply:
https://github.com/cosmos/relayer/tree/master/scripts/nchainz